Specification/BrAPI-Schema/BrAPI-Genotyping/VariantSet.json

```diff
@@ -296,21 +308,14 @@
                     "referencedAttribute": "availableFormats"
```

Comment:
relationshipType is still `many-to-one`, reverse is `many-to-many`

Reply:
A single VarientSet can have multiple formats so one-to-many
A format can be shared between VarientSets, so many-to-many